plynx/web/node.py

```diff
@@ -178,6 +183,19 @@ def post_node(collection: str):
 
         if is_author or is_admin or (is_workflow and can_modify_others_workflows):
             node.save(force=True)
+
+            if is_workflow:
+                if node.schedule:
+                    if not croniter.is_valid(node.schedule):
+                        return (
+                            make_fail_response("Invalid crontab expression found for the schedule."),
+                            HTTPStatus.BAD_REQUEST
+                        )
+
+                    add_celery_periodic_task(str(node._id), node.schedule)
+                else:
+                    delete_celery_periodic_task(str(node._id))
+
         else:
             return make_permission_denied('Only the owners or users with CAN_MODIFY_OTHERS_WORKFLOWS role can save it')
```

Comment:
Try to flatten nested if statements. They can be difficult to read and can make debugging more awkward. 

I have taken a shot at doing this, but I'm not 100% its the exact intention.

```suggestion
        can_edit_workflow = (
            is_author or is_admin or (is_workflow and can_modify_others_workflows)
        )

        if not can_edit_workflow:
            return make_permission_denied('Only the owners or users with CAN_MODIFY_OTHERS_WORKFLOWS role can save it')
            
        node.save(force=True)

        if not is_workflow:
            return

        if node.schedule:
            if not croniter.is_valid(node.schedule):
                return (
                    make_fail_response("Invalid crontab expression found for the schedule."),
                    HTTPStatus.BAD_REQUEST
                )

            add_celery_periodic_task(str(node._id), node.schedule)
        else:
            delete_celery_periodic_task(str(node._id))
```

Reply:
Thanks for that, but this endpoint is used for both workflows and operations. The added piece of code only applies for workflow